app/views/publishers/_instant_donation_modal.html.slim

```diff
@@ -6,6 +6,12 @@
         h2.modal-title = t ".headline"
         p = t ".intro"
         = link_to(t(".continue"), "#", class: 'btn btn-primary', id: "open-banner-button")
+        - if !current_publisher.wallet.default_currency.nil? && !current_publisher.wallet.default_currency.eql?('BAT')
```

Comment:
`- if current_publisher.wallet.default_currency.present? && !current_publisher.wallet.default_currency.eql?('BAT')`

Reply:
Fixed. Should work in Ruby Slim